### DISCUSSION TITLE
Any plans for adding a "Dark mode"?

### DISCUSSION THREAD

**Discussion by AbdelrhmanHamouda** [tremorlabs/tremor-npm#162] [Q&A] [answered]
Hi, 
I like this project a lot to be honest. I like how modern everything feels. One question that comes to mind is that if you have a plan to support an "out of the box" Dark mode

**Comment by christopherkindl** [comment, marked as answer]
Hi @AbdelrhmanHamouda, many thanks for testing our library and your kind words! Dark mode is a feature we want to cover in the medium term. At this stage, we are focused on further improving existing components as well as adding new ones (especially in the area of charts) to allow building as many use cases as possible. 

If you have any further feedback, feel free to share! :)

**Reply by AbdelrhmanHamouda** [reply]
Thank you for providing this information
